src/main/scala/wabisabi/Client.scala

```diff
@@ -300,8 +300,10 @@ class Client(esURL: String) extends Logging {
    * @param type The type name to which the mappings will be applied.
    * @param body The mapping.
    */
-  def putMapping(indices: Seq[String], `type`: String, body: String): Future[Response] = {
-    val req = (url(esURL) / indices.mkString(",") / `type` / "_mapping").setBody(body.getBytes(StandardCharsets.UTF_8))
+  def putMapping(indices: Seq[String], `type`: String, body: String, ignoreConflicts: Boolean = false): Future[Response] = {
+    val req = (url(esURL) / indices.mkString(",") / `type` / "_mapping").
+      setQueryParameters(Map("ignore_conflicts" -> List(ignoreConflicts.toString))).
+      setBody(body.getBytes(StandardCharsets.UTF_8))
```

Comment:
I hunted around and can't find any definitive information on what's considered style-appropriate for Scala when it comes to line-wrapping chained methods. I, however, prefer leading dots. Could you change that here?